next.config.js

```diff
@@ -1,31 +1,29 @@
 const { PHASE_DEVELOPMENT_SERVER } = require("next/constants");
 
 const nextConfig = {
-  output: 'export',
+  output: "export",
   reactStrictMode: true,
   trailingSlash: true,
-  distDir: 'build',
-  transpilePackages: [
-    '@pagopa/mui-italia',
-  ],
+  distDir: ".next",
```

Comment:
Here you are telling that the directory where next builds, will be the same directory where next puts the code for development and it's wrong!

Restore the old version

Reply:
I see the version with .next. Restore the old version